feat: add GraphQL API support (#285)
crowdin/crowdin-api-client-dotnet: README.md

Console.WriteLine(projectSettingsResponse);
```

#### GraphQL API

> This SDK also has support of [GraphQL API](https://support.crowdin.com/developer/graphql-api/) (only for Crowdin Enterprise).

Reviewer: ```suggestion
This SDK also has the support of [GraphQL API](https://support.crowdin.com/developer/graphql-api/).
```
